Altis_Life.Altis/core/functions/fn_keyHandler.sqf

```diff
@@ -44,12 +44,21 @@ if (!(count (actionKeys "User10") isEqualTo 0) && {(inputAction "User10" > 0)})
     true;
 };
 
-if (life_container_active) then {
-    switch (_code) do {
-        //space key
-        case 57: {
-            [] spawn life_fnc_placestorage;
+if (life_container_active) exitwith {
+    //ignore movement actions
+    _allowedMoves = ["MoveForward", "MoveBack", "TurnLeft", "TurnRight", "MoveFastForward", "MoveSlowForward", "turbo", "TurboToggle", "MoveLeft", "MoveRight", "WalkRunTemp", "WalkRunToggle", "AdjustUp", "AdjustDown", "AdjustLeft", "AdjustRight", "Stand", "Crouch", "Prone", "MoveUp", "MoveDown", "LeanLeft", "LeanLeftToggle", "LeanRight", "LeanRightToggle"];
```

Comment:
You should probably put this list in a vertical manner instead of horizontal for readability. It is very big. Also privatize the variable.

Reply:
good idea, will change this